jekyll/_docs/build-image-ubuntu-14.04-XL-922-9410082.md

```diff
@@ -6,6 +6,11 @@ description: ubuntu-14.04-XL-922-9410082
 changefreq: "weekly"
 ---
 
+***Please note that we are planning to make the image depreciated on CircleCI 2.0 in the future.***
+
+The idea of monolithic build image doesn't fix the idea of CircleCI 2.0 quite well and we are currently
```

Comment:
> depreciated

Do you mean deprecated?

> doesn't fix the idea of CircleCI 2.0 

@kimh Do you mean "doesn't fit the idea of CircleCI 2.0"?